fv3core/testing/parallel_translate.py

```diff
@@ -176,13 +176,13 @@ def compute_sequential(self, a, b):
 
 
 class ParallelTranslate2PyState(ParallelTranslate2Py):
-    def compute_parallel(self, inputs, communicator):
+    def compute_parallel(self, inputs):
```

Comment:
Changing the call signature of this can be confusing for people who are used to the previous signature, and when reading other objects using this class or similar parallel translate classes.

Instead of changing the API, could you instead initialize the object inside `compute_parallel`, rather than inside the translate class `init`? It does look like this is what you're actually doing, since while you changed the base class API, the subclass you refactored still takes in `communicator` in its compute. Since these are being called once, it shouldn't matter whether the class gets made on each call or on each init.